conversation_reconstruction_dataflow/conversation_reconstruction/dataflow_main.py

```diff
@@ -11,104 +13,91 @@
 import subprocess
 import json
 from os import path
+import urllib2
+import traceback
 
 import apache_beam as beam
 from apache_beam.options.pipeline_options import PipelineOptions
 from apache_beam.options.pipeline_options import SetupOptions
-from apache_beam.io import ReadFromText
-from apache_beam.io import WriteToText
-from google.cloud import storage
-import traceback
+from apache_beam.io.gcp import bigquery as bigquery_io 
 
+def run(known_args, pipeline_args):
+  """Main entry point; defines and runs the reconstruction pipeline."""
+
+  pipeline_args.extend([
+    '--runner=DataflowRunner',
+    '--project=wikidetox-viz',
+    '--staging_location=gs://wikidetox-viz-dataflow/staging',
+    '--temp_location=gs://wikidetox-viz-dataflow/tmp',
+    '--job_name=yiqing-reconstruction-test',
```

Comment:
Best not to put your name into jobs... if you want to make it have your name, maybe best to get username and concat with the postfix  "-reconstruction-test"?